packages/pixeloven-core/env/src/index.ts

```diff
@@ -1,11 +1,10 @@
 /**
  * Register libraries here
  */
+export * from "./Env";
```

Comment:
We aren't using this anywhere but I have a future cli addon planned that will re-use this. Wanted to keep in there for now.